src/transformers/models/reformer/modeling_reformer.py

```diff
@@ -2392,12 +2440,7 @@ def __init__(self, config):
         self.post_init()
 
     @add_start_docstrings_to_model_forward(REFORMER_INPUTS_DOCSTRING)
-    @add_code_sample_docstrings(
-        processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
-        output_type=SequenceClassifierOutput,
-        config_class=_CONFIG_FOR_DOC,
-    )
+    @replace_return_docstrings(output_type=SequenceClassifierOutput, config_class=_CONFIG_FOR_DOC)
```

Comment:
The reason for using `replace_return_docstrings` here is below:
- I got an error when calling `loss.backward()`. So, I replaced it with simply print loss. (line: 2530)

Reply:
Update:
- adding pad_token to pad inputs. (line: 2509)
- padding max_length=100 to avoid the error. (line: 2510)
- model.train() to call `loss.backward()`. (line: 2526)